### PR TITLE
Feat: update patient and clinical trials parser 

### PR DESCRIPTION
**Pull request by WojciechKusa**
- bump version to `0.0.2`
- patient loader now splits text into sections: past, current and family medical history
- clinical trial text is now a post_init field, not a method
- updates in relevant tests

### CODE REVIEW

File: tests/functional/test_clinical_trials_parser_from_folder.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 HERE = os.path.dirname(os.path.abspath(__file__))
```
Comment (ospitia) [inline tests/functional/test_clinical_trials_parser_from_folder.py]: HERE?

File: CTnlp/patient/__init__.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    conditions: Optional[List[str]] = None
+    current_medical_history: Optional[str] = None
+    past_medical_history: Optional[str] = None
+    family_medical_history: Optional[str] = None
+
+    gender: Gender = Gender.unknown
+    age: Union[int, float, None] = None
+    is_healthy: Optional[bool] = None
+    is_smoker: Optional[bool] = None
+    is_drinker: Optional[bool] = None
```
Comment (ospitia) [inline CTnlp/patient/__init__.py:27]: wow ,so specific 👌

**Review by ospitia — APPROVED** [review]

File: tests/functional/test_clinical_trials_parser_from_folder.py (hunk 1 above)
Comment (WojciechKusa) [reply tests/functional/test_clinical_trials_parser_from_folder.py]: I don' know how should we name the path to the current file directory :D

Comment (WojciechKusa) [reply tests/functional/test_clinical_trials_parser_from_folder.py]: some ideas? :>

File: CTnlp/patient/__init__.py (hunk 2 above)
Comment (WojciechKusa) [reply CTnlp/patient/__init__.py:27]: haha, currently this is empty but maybe we can extend it at some point

File: tests/functional/test_clinical_trials_parser_from_folder.py (hunk 1 above)
Comment (ospitia) [reply tests/functional/test_clinical_trials_parser_from_folder.py]: current file directory sounds better

Comment (WojciechKusa) [reply tests/functional/test_clinical_trials_parser_from_folder.py]: changed, thanks :)

**Pull request merged**